tests/TestCase/I18n/TimeTest.php

```diff
@@ -482,6 +490,40 @@ public function testI18nFormat($class)
     }
 
     /**
+     * test formatting dates taking in account default output timezones.
+     *
+     * @dataProvider classNameProvider
+     * @return void
+     */
+    public function testI18nFormatWithDefaultOutputTimezone($class)
+    {
+        $time = new $class('Thu Jan 14 13:59:28 2010');
+
+        $class::$defaultLocale = 'en-CA';
+        $class::setDefaultOutputTimezone('America/Vancouver');
+
+        $result = $time->i18nFormat();
+        $expected = '1/14/10 5:59 AM';
+        $this->assertTimeFormat($expected, $result);
+
+        $result = $time->i18nFormat(null, 'America/Toronto');
+        $expected = '1/14/10 8:59 AM';
+        $this->assertTimeFormat($expected, $result);
```

Comment:
There should probably be tests for other methods like `nice()`, `__toString()` and `timeAgoInWords()`.